pkg/cloud/vsphere/provisioner/govmomi/utils.go

```diff
@@ -125,3 +141,19 @@ func (pv *Provisioner) GetKubeConfig(cluster *clusterv1.Cluster) (string, error)
 	}
 	return string(secret.Data[constants.KubeConfigSecretData]), nil
 }
+
+func clusterProviderFromProviderConfig(providerConfig clusterv1.ProviderConfig) (*vsphereconfigv1.VsphereClusterProviderConfig, error) {
```

Comment:
There are already util methods in the pkg/cloud/vsphere/utils/utils.go for doing this. (I see you updated them as well)
So, do we need these methods here as well?